mappings/net/minecraft/block/SegmentedFlowerBlock.mapping

```diff
@@ -0,0 +1,6 @@
+CLASS net/minecraft/class_10594 net/minecraft/block/SegmentedFlowerBlock
```

Comment:
Should be named `FlowerbedBlock` to match its model's name and previous yarn name I think